examples/regression/trace/issue527.spthy

```diff
@@ -0,0 +1,59 @@
+theory issue527
+begin
+
```

Comment:
Can you add a small comment explaining the purpose of this file?

Like this:
/*
   Protocol:    Example
   Modeler:    Test
   Date:        January 2012

   Status:      working

   Description of protocol.

*/